src/main/java/io/github/thebusybiscuit/slimefun4/api/events/MultiBlockBreakEvent.java

```diff
@@ -0,0 +1,83 @@
+package io.github.thebusybiscuit.slimefun4.api.events;
+
+import java.util.List;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import javax.annotation.ParametersAreNonnullByDefault;
+
+import org.bukkit.block.Block;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+
+/**
+ * This {@link Event} is called when a {@link Player} breaks multiply blocks.
+ * 
+ * @author TheBusyBiscuit
```

Comment:
That's what I call identity theft :eyes:

Reply:
Oh, sorry xD forgot to change that